test/sys/test_event.rs

```diff
@@ -0,0 +1,19 @@
+use std::time::Duration;
+
+use libc::timespec;
+use nix::sys::event::{kqueue, kevent};
+
+#[test]
+pub fn test_kevent() {
+    let kq = kqueue().unwrap();
+
+    let mut events = Vec::new();
+
+    let timeout = Duration::from_millis(0);
+    assert_eq!(kevent(kq, &[], &mut events, Some(timeout)).unwrap(), 0);
```

Comment:
With the above function call change you can get rid of the `Some(...)` wrapping.